wisper.gemspec

```diff
@@ -24,8 +24,7 @@ Gem::Specification.new do |gem|
     gem.cert_chain  = ['gem-public_cert.pem']
   end
 
-  gem.files         = `git ls-files`.split($/)
+  gem.files         = %w{LICENSE} + Dir.glob("{lib,bin}/**/*")
```

Comment:
Does the gem need wisper.gemspec?

Reply:
You don't actually need it in the gem artifact. You just need it in the repo to generate the artifact.